resources/antares-desktop-fs/gunicorn.py

```diff
@@ -0,0 +1,30 @@
+# Reference: https://github.com/benoitc/gunicorn/blob/master/examples/example_config.py
```

Comment:
Actually, we don't use gunicorn to run the server in desktop mode:
instead, a single uvicorn worker is started.
So we can remove this file :)